packages/web-ui/src/ds/molecules/Chat/Message/index.tsx

```diff
@@ -65,7 +65,7 @@ export function MessageItem({
           className='flex-shrink-0 bg-muted w-1 rounded-lg cursor-pointer hover:bg-primary transition-colors'
           onClick={() => setCollapseMessage(!collapsedMessage)}
         />
-        <div className='flex flex-1 flex-col gap-1'>
+        <div className='flex flex-grow flex-col gap-1 overflow-x-auto'>
```

Comment:
Be aware that this component is also used in shared prompts. Maybe check if it looks ok

Reply:
reviewed and it looks good